status-im/status-react
# chore: separate resources for old and new ui

Refactor as part of migration to status_im2.

created a new resources.cljs in status_im2.

Left initial resources.cljs in status_im folder but removed any references for resources used in status_im2 and quo2.

created new resources/images/mock2 and resources/images/ui2 folder which we can use for images for the new ui implementation.

updated all references to now point to the correct location 👍 

To Test: 
Make sure the app loads without error and just navigate to some pages where images etc are showing.
